src/main/java/mate/academy/spring/config/SecurityConfig.java

```diff
@@ -30,17 +30,17 @@ public void configure(AuthenticationManagerBuilder auth) throws Exception {
     protected void configure(HttpSecurity http) throws Exception {
         http
                 .authorizeRequests()
-                .antMatchers(HttpMethod.POST,"/register").permitAll()
-                .antMatchers(HttpMethod.GET,"/cinema-halls/",
-                        "/movies/", "/movie-sessions/available").hasAnyRole(ADMIN, USER)
-                .antMatchers(HttpMethod.POST,"/cinema-halls/",
-                        "/movies/", "/movie-sessions/").hasRole(ADMIN)
-                .antMatchers(HttpMethod.DELETE,"/movie-sessions/{id}").hasRole(ADMIN)
-                .antMatchers(HttpMethod.PUT,"/movie-sessions/{id}").hasRole(ADMIN)
-                .antMatchers(HttpMethod.GET,"/orders/", "/shopping-carts/by-user").hasRole(USER)
-                .antMatchers(HttpMethod.POST,"/orders/complete").hasRole(USER)
-                .antMatchers(HttpMethod.PUT,"/shopping-carts/movie-sessions").hasRole(USER)
-                .antMatchers(HttpMethod.GET,"/users/by-email").hasRole(ADMIN)
+                .antMatchers(HttpMethod.POST, "/register").permitAll()
+                .antMatchers(HttpMethod.POST,"/cinema-halls",
+                        "/movies", "/movie-sessions").hasRole("ADMIN")
+                .antMatchers(HttpMethod.POST, "/orders/complete").hasRole("USER")
```

Comment:
<img width="550" alt="image" src="https://user-images.githubusercontent.com/58032907/209820310-43de587f-bb36-464f-bc57-eddc7e022088.png">
let's import Role.Type.ADMIN and USER and use ADMIN.name() instead of string literals